panel/lxqtpanel.cpp

```diff
@@ -374,7 +372,7 @@ int LXQtPanel::getReserveDimension()
     return mHidable ? PANEL_HIDE_SIZE : qMax(PANEL_MINIMUM_SIZE, mPanelSize);
 }
 
-void LXQtPanel::setPanelGeometry(bool animate)
+void LXQtPanel::setPanelGeometry(bool animate, bool firstTime)
```

Comment:
I don't get this `firstTime` flag.

Should the initial panel showup be animated or not? Right now I see the animation, but you were trying to disable it before (IIRC).

But no matter if the fist time show should be animated or not, I still think this flag is not necessary.